ZelBack/src/lib/server.js

```diff
@@ -4,8 +4,15 @@ const cors = require('cors');
 const morgan = require('morgan');
 const compression = require('compression');
 
-const expressWs = eWS(express());
-const { app } = expressWs;
+// see https://github.com/HenningM/express-ws/issues/120
+const options = {
+  wsOptions: {
+    maxPayload: 65535,
+  },
+};
+
+const app = express();
+const expressWs = eWS(app, null, options);
```

Comment:
should not this be 
const { app } = expressWs(express(), undefined, options); ??

Reply:
It's the same thing. I personally find the above more readable. (the server option can be underfined or null)